src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1209,7 +1209,13 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         _RestorePointerCursorHandlers(*this, nullptr);
 
         const auto point = args.GetCurrentPoint(*this);
-
+        // GH#10329 - we don't need to handle horizontal scrolls. Only vertical ones.
+        // So filter out the horizontal ones.
+        if (point.Properties().IsHorizontalMouseWheel())

```

Comment:
thatwaseasy.gif

Reply:
Should it call `args.Handled()` though?

Reply:
I suspect that since we are not handling it, we should not mark it handled. Somebody else might want it :smile: